app/views/simulator/edit.html.erb

```diff
@@ -11,12 +11,9 @@
 
 
 <!-- <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css"> -->
-<link rel="stylesheet" href="/css/bootstrap-3.3.7.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
We are using a custom build of CSS. Does the look of the simulator remain the same? I doubt it.

Reply:
Okay, I'll look into it.

Reply:
Looks like I was wrong, @sanctophagus can you confirm once, that all pages look same as before with no errors. I will merge then.

Reply:
@satu0king hey, all the bootstrap files are independent. We can go ahead and merging with this but I would wait for @sanctophagus to confirm once.

Reply:
@armudgal @satu0king  LGTM.